tox.ini

```diff
@@ -50,12 +48,36 @@ commands =
     !universal2: {envdir}/bin/alltests --layer '!Products.PluginIndexes' {posargs:-vc}
 allowlist_externals = *
 
+[testenv:setuptools-latest]
+basepython = python3
+deps =
+    git+https://github.com/pypa/setuptools.git\#egg=setuptools
+    zc.buildout >= 3.1
+    wheel > 0.37
+    cffi >= 1.17.0rc1
+    # The universal2 environment is for Python on macOS built with
+    # universal2 mode instead of architecture-specific. These dependencies
+    # must be installed separately, zc.buildout is incompatible with the
+    # universal2 mode and cannot install them itself.
+    universal2: -c {toxinidir}/constraints.txt
+    universal2: zope.interface
+    universal2: zope.security
+    universal2: zope.container
+    universal2: Persistence
+    universal2: Acquisition
+    universal2: AccessControl
+    universal2: zodbpickle
+    universal2: charset_normalizer
+    universal2: MarkupSafe
+    universal2: zope.testrunner
+
 
 [testenv:release-check]
 description = ensure that the distribution is ready to release
 basepython = python3
 skip_install = true
 deps =
+    setuptools <74
```

Comment:
Usually, we have blanks around comparison operators

Reply:
"Whitespace between a conditional operator and the following version identifier is optional, as is the whitespace around the commas.", see https://packaging.python.org/en/latest/specifications/version-specifiers/#id5

Reply:
The definition for that specification is in zope.meta, not in this package.

Reply:
I have cleaned up the spacing in zope.meta and applied it here.